drizztsaga/language/polish/bg/main.tra

```diff
@@ -1042,15 +1042,16 @@ Ten toporek zosta
 
 PARAMETRY:
 
+Specjalne:
```

Comment:
This one is incorrect for oBG2 format in Polish. I will revert this change along with my future fixes. Polish translation has got a big update in this version, so no need to update existent entries.

Reply:
this has been revert